packages/clay-card/src/Body.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import React from 'react';
+
+import classNames from 'classnames';
+
+import Context from './Context';
+
+export interface CardBodyProps extends React.HTMLAttributes<HTMLDivElement> {
+	children:
+		| React.ReactElement<HTMLElement>[]
+		| React.ReactElement<HTMLElement>;
+}
+
+export type CardBodyType = React.FunctionComponent<CardBodyProps>;
+
+export const Body: CardBodyType = ({children, className, ...otherProps}) => {
+	const {displayType} = React.useContext(Context);
+
+	const isCardType = {
+		directory: displayType === 'directory',
+		horizontalInteractive: displayType === 'horizontal interactive',
+		interactive: displayType === 'interactive',
+	};
+
+	const TagName =
+		isCardType.interactive || isCardType.horizontalInteractive
+			? 'span'
+			: 'div';
+
+	if (isCardType.directory) {
```

Comment:
```suggestion
	if (displayType === 'directory') {
```

Then we can remove `isCardType` from here.

Reply:
Solved.